joblib/testing.py

```diff
@@ -51,6 +68,24 @@ def assert_raises_regex(expected_exception, expected_regexp,
                                      expected_exception(expected_regexp))
 
 
+try:
```

Comment:
Is this required ? Joblib is already tested against python 2.6 and it doesn't need this.